flink-tools/src/main/java/io/pravega/flinktools/StreamToParquetFileJob.java

```diff
@@ -116,10 +120,8 @@ public void run() {
                 toOutput.print("output");
             }
 
-            // You need to overwrite the flink-parquet as a shadowJar along with withCompressionCodec method calling
-            // when plan to use CompressionCodecName other than UNCOMPRESSED
             final StreamingFileSink<GenericRecord> sink = StreamingFileSink
-                    .forBulkFormat(new Path(outputFilePath), ParquetAvroWriters.forGenericRecord(outputSchema).withCompressionCodec(CompressionCodecName.SNAPPY))
```

Comment:
Can you clarify how you reimlement the compression after  `.withCompressionCodec(CompressionCodecName.SNAPPY))` is deleted?

Reply:
Sorry I missed the compression part, I have added the writers for supporting different compression methods.
- Running jobs
![image](https://user-images.githubusercontent.com/39532976/205016020-050250cb-ebb2-46c9-ac60-bee789af1d48.png)
- Compressed parquet files
![image](https://user-images.githubusercontent.com/39532976/205016829-91c3fc3f-529c-4828-b74a-79b2d0fedc14.png)